spec/airport.spec.js

```diff
@@ -144,14 +144,13 @@ console.log(`Test 6: The airport can confirm that a plane is not in the airportP
 
 //Arrange  
```

Comment:
I can now use the describe and it functions from Jasmine to write tests more clearly and efficiently.